addons/mass_mailing/models/utm_campaign.py

```diff
@@ -69,12 +61,6 @@ def _compute_mailing_mail_count(self):
             campaign.mailing_mail_count = sum(mapped_data.get(campaign._origin.id or campaign.id, []))
             campaign.ab_testing_mailings_count = sum(ab_testing_mapped_data.get(campaign._origin.id or campaign.id, []))
 
-    @api.constrains('ab_testing_total_pc', 'ab_testing_completed')
```

Comment:
as discussed IRL, can we move this constrains on the mailing ? (`ab_testing_pc <= 100`)

Reply:
Oh actually this should already be the case since there's an _sql_constraint on that. I believe it would be triggered at the same time?